lib/koans/15_processes.ex

```diff
@@ -17,9 +17,20 @@ defmodule Processes do
     assert is_pid(self()) == ___
   end
 
-  koan "New processes are spawned functions" do
+  koan "Like our current process, new processes are spawned functions" do
     pid = spawn(fn -> nil end)
 
+    assert is_pid(pid) == ___
```

Comment:
Nevermind my recommendation about `is_pid(self)`. I read the lesson in context and see that it's covered just before this.